### DISCUSSION TITLE
Why doesn't Rowy show all results?

### DISCUSSION THREAD

**Discussion by StuttEx** [AntlerVC/firetable#1188] [Support / Q&A] [answered]
<img width="234" alt="Bildschirm­foto 2023-03-30 um 10 35 09" src="https://user-images.githubusercontent.com/127051951/228778907-c297bbda-9981-40f8-9f77-7c7242df5b7f.png">
It's weird, confusing, misleading and unnecessary. Why does Rowy filter the rows?
When I hit refresh, I see for a millisecond that all rows are there – but rowy hides them and tells me it only loads X of Y rows. But it already loaded them – and more important – I need them!
The rows are there and created by a rowy webhook? What's the problem? I granted literal unlimited access to viewer, editor admin etc. activated auditing if this plays any role.
I'm on the PRO plan by the way.
Greetings
Pascal

**Comment by shamsmosowi** [comment, marked as answer]
you might sorting by column with empty values in some(4) of your documents

**Reply by StuttEx** [reply]
Amazing Dude, this was it. I unknowingly hat a sorting in a field with empty values in these rows.

But Rowy, you could improve the UX, somewhere displaying a bit more visible that a sorting is on.
Nevertheless – you saved my day Dude, @shamsmosowi
